alpha/lib/model/category.php

```diff
@@ -1755,4 +1780,50 @@ public function indexCategoryInheritedTree()
 	{
 		kEventsManager::raiseEventDeferred(new kObjectReadyForIndexInheritedTreeEvent($this));
 	}
+
+	/**
+	 * @param $fullIds
+	 * @return array
+	 */
+	private static function getCategoriesStartWithIds($fullIds)
```

Comment:
Please rename to getDescendantCategoriesIds or other functional name, also remove the argument and calculate the full ids string inside the method instead of validating the input.